<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewElfParser</class>
 <widget class="QWidget" name="ViewElfParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="__path_to_elf">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #F9F9F9; border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Загрузите elf файл для анализа...</string>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="__tab_widget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="__w_elf_header">
         <attribute name="title">
          <string>Elf header</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="__table_elf_header"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="__w_elf_sections_headers">
         <attribute name="title">
          <string>Sections Headers</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="__table_section_header"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="__w_program_headers">
         <attribute name="title">
          <string>Program headers</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableWidget" name="__table_program_header"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="__symtab">
         <attribute name="title">
          <string>.symtab</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="__table_sym_tab"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="__dynsym">
         <attribute name="title">
          <string>.dynsym</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="__table_dynsym"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="__b_load_elf_file">
     <property name="text">
      <string>Загрузить elf файл</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
